pkg/volume/csi/csi_plugin.go

```diff
@@ -207,7 +207,8 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
 		csiClient := host.GetKubeClient()
 		if csiClient == nil {
-			klog.Warning("The client for CSI Resources is not available, skipping informer initialization")
+			klog.Error(log("failed to get a kubernetes client"))
```

Comment:
I don't think we need this log message. We can leave it up to the callers to log it

Reply:
Ok